app/features/my-rewards/rewards-card-mobile.tsx

```diff
@@ -55,28 +54,16 @@ function RewardCard({
       <div>
         {showReward ? (
           <RewardBadge
-            amount={fromTokenAmount(contractReward[0].toString())}
+            amount={fromTokenAmount(contractReward.paymentTokenAmount.toString(), 3)}
             token={token?.symbol ?? "Unknown Token"}
-            rMETRIC={contractReward[1].toNumber()}
+            rMETRIC={contractReward.reputationTokenAmount.toNumber()}
           />
         ) : (
           <span>--</span>
         )}
       </div>
       <div>Submitted</div>
       <p className="text-black">{fromNow(reward.createdAtBlockTimestamp)} </p>
-      <div>Rewarded</div>
-      <div className="text-black" color="dark.3">
```

Comment:
Isn't this enforcement expiration? Did we decide to remove?

Reply:
oh hmm good question. I was thinking there's no way to tell when someone was rewarded at the moment, so removing for MVP. I'll make a follow up ticket.